compiler/hash-semantics/src/new/passes/inference/mod.rs

```diff
@@ -29,7 +29,7 @@ impl_access_to_tc_env!(InferencePass<'_>);
 
 /// The maximum number of hole-filling iterations to perform when inferring a
 /// term, before giving up and reporting an error.
-const MAX_INFER_ITERATIONS: usize = 20;
+const MAX_INFER_ITERATIONS: usize = 2;
```

Comment:
It would be nice to comment on why exactly "2", and maybe why it's not valid after that?

Reply:
And, it could be made configurable through command line argument i.e. `-Cinference-passes=<?>` with it defaulting to `2`.

Reply:
Yeah currently it's just for testing, I will add an option for it later.

Reply:
also there's nothing special about 2, it's currently just a random value

Reply:
Maybe leave it as a `@@Todo`?